nodejs-jquery/nodejs-jquery.spec

```diff
@@ -0,0 +1,57 @@
+%{?scl:%scl_package nodejs-%{npm_name}}
+%{!?scl:%global pkg_name %{name}}
+
+%global npm_name jquery
+
+Name: %{?scl_prefix}nodejs-%{npm_name}
+Version: 1.11.3
+Release: 1%{?dist}
+Summary: JavaScript library for DOM operations
+License: MIT
+Group: Development/Libraries
+URL: https://github.com/jquery/jquery.git
+Source0: http://registry.npmjs.org/jquery/-/jquery-1.11.3.tgz
+BuildRoot: %{_tmppath}/%{name}-%{version}-%{release}-root-%(%{__id_u} -n)
+Requires: nodejs(engine)
```

Comment:
Yes, but that sounds like correct behaviour to prefer the newer package if it exists.  Adding the SCL prefix to avoid that doesn't help on Fedora, nor does it prevent file conflicts if the package installs into the system root, not the SCL root.

If you want to build the package with an SCL prefix and into the SCL root then the `tfm` SCL should depend on a NodeJS SCL so the various %nodejs* macros get redefined to the right location (https://github.com/theforeman/foreman-packaging/compare/rpm/develop...domcleal:rpm/develop-execjs-nodejs should do this).